### ISSUE TITLE
v6.4.1: Segmentation fault when restarting Tribler

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#1116]
After I edited the configuration (nickname, seeding, bandwidth) it offered me to restart Tribler. But instead of restarting it crashed:

```
GUI closing
GUI closed
./tribler.sh: line 43: 10946 Segmentation fault      python Tribler/Main/tribler.py
```

**Comment by whirm** [comment]
@vi does it crash every time you try starting it? 
if you can reproduce it, could you enable core dumping (```ulimit -c unlimited```) and send me the core file?

Thanks!

**Comment by vi** [comment]
http://vi-server.org/pub/tribler_core.7z

**Comment by whirm** [comment]
Hm, I don't have any i386 system handy ATM.

@vi Could you load the core into gdb with ```gdb python core```, run ```bt``` at the gdb prompt and paste the whole output from gdb? (including the startup messages)

Thanks!

**Comment by vi** [comment]
Result of `thread apply all bt full`:

http://vsodo.vi-server.org/pub/tribler_backtrace.txt.7z
http://vi-server.org/pub/tribler_backtrace.txt.7z

(two links to the same file)

**Comment by whirm** [comment]
Thanks!

**Comment by whirm** [comment]
Found it, @LipuFei is sending a PR as I write.

**Comment by LipuFei** [comment]
Fixed through #1146
